application/src/main/java/org/opentripplanner/updater/trip/gtfs/GtfsRealTimeTripUpdateAdapter.java

```diff
@@ -771,14 +833,15 @@ private boolean routeExists(String feedId, TripDescriptor tripDescriptor) {
    * @param realTimeState     real-time state of new trip
    * @return empty Result if successful or one containing an error
    */
-  private Result<UpdateSuccess, UpdateError> addTripToGraphAndBuffer(
+  private Result<UpdateSuccess, UpdateError> addNewOrReplacementTripToGraphAndBuffer(
```

Comment:
We don't add transit entities to the graph any more (haven't done so for years) and the buffer is no longer called that. For that reason, please call this method add `addNewOrReplacementTripToSnapshot`